adhocteam/Head-Start-TTADP
# Cm 280 delete files

**Description of change**
Adds support to delete files.

**How to test**
1. Pull down changes.
2. Upload files.
3. Verify they are present in minio and the db.
4. click on the trash can icon next to the file
5. verify they are gone from minio and the db

**Issue(s)**
* https://github.com/HHS/Head-Start-TTADP/issues/280

**Checklist**

    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified
